homework/01LessonOne.js

```diff
@@ -17,27 +17,27 @@ var myUndefined;
 //
 // What is the difference between null and undefined?
 
-var myNull;
+var myNull=null;
 
 // Problem 2:
 // Let's create a new variable named myTrue and give it the value true.
 
-var myTrue;
+var myTrue=true;
 
 // Problem 3:
 // Let's create a new variable named myFalse and give it the value false.
 
-var myFalse;
+var myFalse=false;
 
 // Problem 4:
 // Let's create a new variable named myNumber. Assign it a number.
 
-var myNumber;
+var myNumber=27;
 
 // Problem 5:
 // Let's create a new variable named myString. Assign it a string.
 
-var myString;
+var myString="Hola";
```

Comment:
Gracias

Thanks for you feedback. Travis,

It is better to say "Excelente seleccion!" in Spanish "Escogida" could have
another connotation and this would not  be so good.

On Sat, Jul 30, 2016 at 5:44 PM, Travis Jefferson <notifications@github.com>
wrote:

> In homework/01LessonOne.js
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/70#discussion_r72895238>
> :
>
> >
> >  // Problem 5:
> >  // Let's create a new variable named myString. Assign it a string.
> >
> > -var myString;
> > +var myString="Hola";
>
> Escogida excelente!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/AustinCodingAcademy/intermediate-workbook/pull/70/files/3aefc0cd52973e7422557262f44dde763d2d91db#r72895238>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASRWl-LPiGAPm3s4qaGUONTtN2FDniYeks5qa9PTgaJpZM4JUu_9>
> .
>